src/components/PostComment.jsx

```diff
@@ -0,0 +1,60 @@
+import React, { useContext, useEffect, useState } from "react";
+import { postCommentByArticle_id } from "../utils/api";
+import { UserContext} from "../contexts/User";
+
+const PostComment = (params) => {
+    const { commentList, setCommentList, article_id } = params
+    const {user, setUser} = useContext(UserContext)
+    const [isLoading, setIsLoading] = useState(false)
+   
+    const[newCommnet, setNewCommnet] = useState('')
+    const[postError, setPostError] = useState(false)
+    const[emptyComment, setEmptyComment] = useState(false)
+ 
+    const handlePost = (event) => {
+        console.log(newCommnet,"comments before post")
+        event.preventDefault()
+     
+        if(newCommnet.length !==0){
+            postCommentByArticle_id(article_id, user, newCommnet)
+            .then((response)=>{
+                setCommentList((curr)=>{
+                    return [response, ...curr]
+                
+                })
+                setNewCommnet('')
+            })
```

Comment:
Very good :)

Reply:
Thank you. Just added post loading and post successful message as well.